randovania/games/prime1/json_data/header.json

```diff
@@ -1487,16 +1487,7 @@
                                 "items": []
                             }
                         },
-                        "lock": {
-                            "lock_type": "front-blast-back-impossible",
-                            "requirement": {
-                                "type": "and",
-                                "data": {
-                                    "comment": null,
-                                    "items": []
-                                }
-                            }
-                        }
+                        "lock": null
```

Comment:
this will break door lock rando entirely. is this part of the change necessary to fix the issue?

Reply:
Yes. Is this what "free from the back" is for?

Reply:
I see the issue. this is actually going to require a small refactor, then. I'll whip that up today so that you can fix this